app/steps/introduction_contact_information.rb

```diff
@@ -1,36 +1,27 @@
 # frozen_string_literal: true
 
 class IntroductionContactInformation < Step
```

Comment:
changing the name of this requires also changing the name of the controller and views. do you think we should do it now or wait?